clang/lib/Driver/Driver.cpp

```diff
@@ -4787,7 +4787,7 @@ class OffloadingActionBuilder final {
 
     /// The SYCL actions for the current input.
     /// One action per triple/boundarch.
-    SmallVector<Action *, 4> SYCLDeviceActions;
+    ActionList SYCLDeviceActions;
```

Comment:
`ActionList` is actually `typedef SmallVector<Action*, 3> ActionList;`
What difference does it make vs using `SmallVector<Action *, 4>`

Reply:
No difference really - just looks cleaner.

Reply:
Just a nit: Might be a little less confusing if we used the same usage everywhere.

Thanks

Reply:
There is other `ActionList` usage in this area, so that is why I switched here to try and be more consistent.